templates/.rubocop.yml

```diff
@@ -0,0 +1,25 @@
+Encoding:
+  Enabled: false
+
+Documentation:
+  Enabled: false
+
+ClassAndModuleChildren:
+  Enabled: false
+
+ClassLength:
+  Enabled: false
+
+MethodLength:
+  Enabled: false
+
+DoubleNegation:
+  Enabled: false
+
+AllCops:
+  Exclude:
+    - "bin/**/*"
+    - "db/**/*"
+    - "config/**/*"
+    - "Rakefile"
+    - "spec/i18n_spec.rb"
```

Comment:
* `i18n_spec` is copied from a dependency gem of suspenders, so we don't control it
* `Rakefile` by default in rails has a line that's too long
* `config/**/*` has all kinds of violations by default in Rails